README.md

```diff
@@ -120,3 +120,8 @@ Usage
 -----
 
 To be written.
+
+Functions
+---------
+
+All suported spacial functions can be found in the [functions.md](functions.md) file.
```

Comment:
IMO the function list is not that big, and would be better suited in the README directly?